docs/NETWORK-MODELS.md

```diff
@@ -0,0 +1,41 @@
+# Network Models
+
+What are the network models that EVE supports for Edge Container workloads, and how are they implemented?
+
+The first half of this document describes _what_ those network models are and how they impact workload communications. The second half describes _how_ those models are implemented.
+
+## Models
+
+Fundamentally, each Edge Container workload needs to communicate with services outside itself. Those can be one or more of:
+
+* other Edge Container workloads on the same device
+* the network to which the device is attached
+* other Edge Container workloads on one or more other devices
+* a network (VPC) in the public cloud
+* no networking at all
+
+It is possible to define all network requirements using communications permissions model. For example:
+
+* workload-to-workload: mesh (between devices) or switch (same device)
+* workload-to-cloud: VPN
+* workload-to-host-network-NAT: switche
+* workload-to-host-network-transparent: bridge
+
```

Comment:
Again, we really shouldn't be confusing our readers with anything that has to do with the host. Host is completely invisible to edge containers.